html/pfappserver/root/src/views/Configuration/domains/_components/index.js

```diff
@@ -23,10 +20,10 @@ export {
   BaseFormGroupInput                  as FormGroupAdServer,
   BaseFormGroupInput                  as FormGroupDnsServers,
   BaseFormGroupInput                  as FormGroupOu,
-  BaseFormGroupToggle                 as FormGroupNtlmv2Only,
-  BaseFormGroupToggle                 as FormGroupRegistration,
+  BaseFormGroupSwitch                 as FormGroupNtlmv2Only,
+  BaseFormGroupSwitch                 as FormGroupRegistration,
 
-  BaseFormGroupToggleDisabledEnabled  as FormGroupNtlmCache,
+  BaseFormGroupSwitch                 as FormGroupNtlmCache,
```

Comment:
These named exports is not a good idea. It makes it harder to refactor and read, since you don't know what component it actually is until you look at this reference